contracts/sfc/SFC.sol

```diff
@@ -733,21 +782,26 @@ contract SFC is Initializable, Ownable, Version {
         return rewards;
     }
 
+    // Burn FTM tokens
+    // The tokens are sent to the zero address
     function _burnFTM(uint256 amount) internal {
         if (amount != 0) {
             payable(address(0)).transfer(amount);
             emit BurntFTM(amount);
         }
     }
 
+    // Get epoch end time
     function epochEndTime(uint256 epoch) internal view returns (uint256) {
         return getEpochSnapshot[epoch].endTime;
     }
 
+    // Check if an address is redirected
     function _redirected(address addr) internal view returns (bool) {
         return getRedirection[addr] != address(0);
     }
 
+    // Get receiver address and check for redirection
```

Comment:
Get address which should receive rewards and withdrawn stake for the given delegator. The delegator is usually the receiver, unless a redirection is created.